pom.xml

```diff
@@ -131,6 +131,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>com.jamesmurty.utils</groupId>
```

Comment:
Should be updated here: https://github.com/iterate-ch/jets3t/blob/master/pom.xml#L319